3p/integration.js

```diff
@@ -297,6 +297,7 @@ import {yieldbot} from '#ads/vendors/yieldbot';
 import {yieldmo} from '#ads/vendors/yieldmo';
 import {yieldone} from '#ads/vendors/yieldone';
 import {yieldpro} from '#ads/vendors/yieldpro';
+import {yobee} from '#ads/vendors/yobee';
```

Comment:
Please revert changes to 3p/integration.js It is only intended for legacy ad networks. You do not need it.

Reply:
Thanks. Fixed.